addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SpringActuatorScanRule.java

```diff
@@ -133,14 +135,15 @@ public void scan() {
                 if (testMsg == null) {
                     continue;
                 }
+                lastTestMsg = testMsg;
                 String contentType = testMsg.getResponseHeader().getNormalisedContentTypeValue();
                 if (contentType != null && isPage200(testMsg)) {
                     String responseBody = testMsg.getResponseBody().toString();
                     boolean matches =
                             CONTENT_TYPE.matcher(contentType).find()
                                     && JSON_PAYLOAD.matcher(responseBody).find();
                     if (matches) {
-                        raiseAlert(testMsg, Alert.CONFIDENCE_MEDIUM, getRisk());
+                        createAlert(testMsg, Alert.CONFIDENCE_MEDIUM, getRisk());
```

Comment:
I'd suggest setting the message here. Then raising the alert (which you seem to have forgotten 😉 )

ex: createAlert(....).setMessage(testMsg).raise();

Instead of passing the message in I'd suggest just passing the evidence. Then use a literal evidence string for the example.

Reply:
Then there's also no need for `latestTestMsg`